data/core/doc/init.lua

```diff
@@ -103,7 +103,7 @@ function Doc:save(filename, abs_filename)
   else
     assert(self.filename or abs_filename, "calling save on unnamed doc without absolute path")
   end
-  local fp = assert(io.open(filename, "wb"))
+  local fp = assert( io.open(abs_filename, "wb") )
```

Comment:
spaces in assert? this pattern doesn't patch whats done in the same function

Reply:
Good point. Fixed.